source/connect/tls.txt

```diff
@@ -0,0 +1,267 @@
+.. _php-tls:
```

Comment:
Q: Not seeing this page linked to from anywhere else, nor is it included in the sidebar. Do we have a `connect` page tree landing yet?

Reply:
saving that for last, when all pages are done